docs/lib/auth/fragments/native_common/signin/common.md

```diff
@@ -117,3 +117,9 @@ As a result, for testing purposes, you'll at least need an input field where you
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signin/50_multi_factor_confirm_signin.md"></inline-fragment>
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signin/50_multi_factor_confirm_signin.md"></inline-fragment>
 <inline-fragment platform="flutter" src="~/lib/auth/fragments/flutter/signin/50_multi_factor_confirm_signin.md"></inline-fragment>
+
+## Check user sign-in state
+
+Invoke the following api to determine whether the current user is signed in or not.
```

Comment:
The main customer use case we want to document here is how clients should determine on app start whether to show a sign in screen or their primary application screen which requires authenticated content.  That's not necessarily the same as whether the current user is signed in or not (if the refresh token expires, the end user might still be signed in according to fetchAuthSession().isSignedIn, but they are no longer authenticated because the refresh token is expired.

```suggestion
Use the following method to check if the current user is authenticated.   This is the recommended approach for determining whether you should display a sign in screen when your application starts.
```